packages/forklift-console-plugin/src/modules/Providers/views/details/components/CredentialsSection/components/edit/OpenstackCredentialsEditFormGroups/ApplicationCredentialNameSecretFieldsFormGroup.tsx

```diff
@@ -127,8 +135,10 @@ export const ApplicationCredentialNameSecretFieldsFormGroup: React.FC<EditCompon
         label={t('Username')}
         isRequired
         fieldId="username"
-        helperText={t('OpenStack REST API user name.')}
-        helperTextInvalid={t('Invalid username.')}
+        helperText={t('A user name for connecting to the OpenStack Identity (Keystone) endpoint.')}
+        helperTextInvalid={t(
+          "Error: The format of the provided user name is invalid. Ensure the user name doesn't include whitespace characters.",
+        )}
```

Comment:
Either "user name" or "username" is OK -- but please use the same for the Ui and the messages, Help texts, etc.